src/handlers/app_listview.rs

```diff
@@ -0,0 +1,45 @@
+use crate::ui::AppWindow; // Import the re-exported AppWindow type
+// use crate::ui_events::UIEvent; // Import the UIEvent enum
+use slint::Model;
+use slint::ModelRc;
+use slint::SharedString;
+use slint::VecModel;
+use slint::ComponentHandle;
+use std::rc::Rc;
+
+pub enum AppListviewEvent {
+    AddItem(SharedString),
+    // Add more events here as needed
```

Comment:
In here we can list all the event types we want AppListview to be able to handle...